Automattic/woocommerce-payments
# Temporarily use WC 7.7 in the E2E tests wcpay-subscriptions tests matrix

Fixes #7451 

#### Changes proposed in this Pull Request

WCS 5.5 is not compatible with WC 7.6, the minimum WC it supports is 7.7

- As a quick fix, replace WC `7.6.0` by `7.7.0` for the `wcpay-subscriptions-tests` job in the E2E tests workflow

#### Testing instructions

* Trigger a run for the E2E tests based on this branch and confirm it no longer fails due to WC 7.6

Test run I triggered: https://github.com/Automattic/woocommerce-payments/actions/runs/6484728285

-------------------


**Post merge**
